### PR TITLE
fixed CQ for bitmessageqt.addressvalidator module

### PR DESCRIPTION
**Pull request by coolguy-cell**
Flake8, Pycodestyle, pylint fixes for bitmessageqt.addressvalidator module

### CODE REVIEW

File: src/bitmessageqt/addressvalidator.py [hunk 1]
```diff
@@ -105,9 +130,15 @@ def validate(self, s, pos):
 
             # version too high
             if decodeAddress(address)[0] == 'versiontoohigh':
-                self.setError(_translate("AddressValidator", "Address too new. Although that Bitmessage address might be valid, its version number is too new for us to handle. Perhaps you need to upgrade Bitmessage."))
+                self.setError(
+                    _translate(
+                        "AddressValidator",
+                        "Address too new. Although that Bitmessage \
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:136]: I'd just put "" into each line. This has too many spaces.

File: src/bitmessageqt/addressvalidator.py [hunk 2]
```diff
@@ -1,14 +1,29 @@
...
-from account import getSortedAccounts
 from queues import apiAddressGeneratorReturnQueue, addressGeneratorQueue
 from tr import _translate
-from utils import str_chan
 
-class AddressPassPhraseValidatorMixin():
-    def setParams(self, passPhraseObject=None, addressObject=None, feedBackObject=None, buttonBox=None, addressMandatory=True):
+
+class AddressPassPhraseValidatorMixin(object):
+    """Address passphrase validator class for QT UI"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:17]: `"""Bitmessage address or passphrase validator class for QT UI"""`

File: src/bitmessageqt/addressvalidator.py [hunk 3]
```diff
@@ -1,14 +1,29 @@
...
+    def setParams(
+            self,
+            passPhraseObject=None,
+            addressObject=None,
+            feedBackObject=None,
+            buttonBox=None,
+            addressMandatory=True,
+    ):
+        """setting parameters for pasphrase, button,address etc"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:26]: `"""Initialisation"""`

File: src/bitmessageqt/addressvalidator.py [hunk 4]
```diff
@@ -19,6 +34,7 @@ def setParams(self, passPhraseObject=None, addressObject=None, feedBackObject=No
         self.okButtonLabel = self.buttonBox.button(QtGui.QDialogButtonBox.Ok).text()
 
     def setError(self, string):
+        """setting the error based on AddressValidation"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:37]: `"""Indicate that the validation is pending or failed"""`

File: src/bitmessageqt/addressvalidator.py [hunk 5]
```diff
@@ -29,11 +45,14 @@ def setError(self, string):
...
-                self.buttonBox.button(QtGui.QDialogButtonBox.Ok).setText(_translate("AddressValidator", "Invalid"))
+                self.buttonBox.button(QtGui.QDialogButtonBox.Ok).setText(
+                    _translate("AddressValidator", "Invalid"))
             else:
-                self.buttonBox.button(QtGui.QDialogButtonBox.Ok).setText(_translate("AddressValidator", "Validating..."))
+                self.buttonBox.button(QtGui.QDialogButtonBox.Ok).setText(
+                    _translate("AddressValidator", "Validating..."))
 
     def setOK(self, string):
+        """setting ok button label"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:55]: `"""Indicate that the validation succeeded"""`

File: src/bitmessageqt/addressvalidator.py [hunk 6]
```diff
@@ -46,12 +65,13 @@ def setOK(self, string):
             self.buttonBox.button(QtGui.QDialogButtonBox.Ok).setText(self.okButtonLabel)
 
     def checkQueue(self):
+        """checking Queue"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:68]: `"""Validator queue loop"""`

File: src/bitmessageqt/addressvalidator.py [hunk 7]
```diff
@@ -60,25 +80,30 @@ def checkQueue(self):
...
+            self.setError(
+                _translate(
+                    "AddressValidator",
+                    "Although the Bitmessage address you \
+                    entered was valid, it doesn\'t match the chan name."))
             return (QtGui.QValidator.Intermediate, 0)
         self.setOK(_translate("MainWindow", "Passphrase and address appear to be valid."))
 
     def returnValid(self):
+        """method to check for validate"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:100]: `"""Return whether the validation was successful"""`

File: src/bitmessageqt/addressvalidator.py [hunk 8]
```diff
@@ -60,25 +80,30 @@ def checkQueue(self):
...
         if self.isValid:
             return QtGui.QValidator.Acceptable
-        else:
-            return QtGui.QValidator.Intermediate
+        return QtGui.QValidator.Intermediate
 
     def validate(self, s, pos):
+        """method check address validation"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:106]: `"""Top level validator method"""`

File: src/bitmessageqt/addressvalidator.py [hunk 9]
```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
...
+                 str_chan + ' ' + str(passPhrase), passPhrase, False))
 
         if self.buttonBox.button(QtGui.QDialogButtonBox.Ok).hasFocus():
             return (self.returnValid(), pos)
-        else:
-            return (QtGui.QValidator.Intermediate, pos)
+        return (QtGui.QValidator.Intermediate, pos)
 
     def checkData(self):
+        """method to check for data"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:165]: `"""Validator QT signal interface"""`

**Comment by coolguy-cell** [comment]
sure

**Comment by coolguy-cell** [comment]
Fixed suggested changes.

File: src/bitmessageqt/addressvalidator.py [hunk 10]
```diff
@@ -1,14 +1,29 @@
-from PyQt4 import QtGui
+"""
+Address validator module.
+"""
+# pylint: disable=too-many-branches,relative-import,too-many-arguments,unused-argument
+
 from Queue import Empty
+from PyQt4 import QtGui
+from utils import str_chan
+from account import getSortedAccounts
```
Comment (g1itch) [inline src/bitmessageqt/addressvalidator.py:9]: Please follow PEP8. You mixed local and library imports.
https://www.python.org/dev/peps/pep-0008/#imports

Comment (coolguy-cell) [reply src/bitmessageqt/addressvalidator.py:9]: @g1itch pylint was complaining for 
Imports from package are not grouped
that' why i changed this.

File: src/bitmessageqt/addressvalidator.py [hunk 11]
```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
...
+                 str_chan + ' ' + str(passPhrase), passPhrase, False))
 
         if self.buttonBox.button(QtGui.QDialogButtonBox.Ok).hasFocus():
             return (self.returnValid(), pos)
-        else:
-            return (QtGui.QValidator.Intermediate, pos)
+        return (QtGui.QValidator.Intermediate, pos)
 
     def checkData(self):
+        """Validator QT signal interface"""
```
Comment (g1itch) [inline src/bitmessageqt/addressvalidator.py:165]: Qt is not QT. QT is Quick Time.

File: src/bitmessageqt/addressvalidator.py (hunk 10 above)
Comment (g1itch) [reply src/bitmessageqt/addressvalidator.py:9]: so your pylint is buggy

Comment (PeterSurda) [reply src/bitmessageqt/addressvalidator.py:9]: utils should go at the end

File: src/bitmessageqt/addressvalidator.py [hunk 12]
```diff
@@ -1,14 +1,29 @@
...
-from account import getSortedAccounts
 from queues import apiAddressGeneratorReturnQueue, addressGeneratorQueue
 from tr import _translate
-from utils import str_chan
 
-class AddressPassPhraseValidatorMixin():
-    def setParams(self, passPhraseObject=None, addressObject=None, feedBackObject=None, buttonBox=None, addressMandatory=True):
+
+class AddressPassPhraseValidatorMixin(object):
+    """Bitmessage address or passphrase validator class for QT UI"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:17]: yea as g1itch said it should be `Qt` not `QT`, my bad

File: src/bitmessageqt/addressvalidator.py (hunk 11 above)
Comment (PeterSurda) [reply src/bitmessageqt/addressvalidator.py:165]: yea

File: src/bitmessageqt/addressvalidator.py [hunk 13]
```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
...
-            return (QtGui.QValidator.Intermediate, pos)
+        return (QtGui.QValidator.Intermediate, pos)
 
     def checkData(self):
+        """Validator QT signal interface"""
         return self.validate("", 0)
 
+
 class AddressValidator(QtGui.QValidator, AddressPassPhraseValidatorMixin):
+    """AddressValidator class for QT UI"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:170]: same here

File: src/bitmessageqt/addressvalidator.py [hunk 14]
```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
...
     def __init__(self, parent=None, passPhraseObject=None, feedBackObject=None, buttonBox=None, addressMandatory=True):
         super(AddressValidator, self).__init__(parent)
         self.setParams(passPhraseObject, parent, feedBackObject, buttonBox, addressMandatory)
 
 
 class PassPhraseValidator(QtGui.QValidator, AddressPassPhraseValidatorMixin):
+    """PassPhraseValidator class for QT UI"""
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:177]: same here

**Comment by coolguy-cell** [comment]
Fixed.

File: src/bitmessageqt/addressvalidator.py (hunk 10 above)
Comment (PeterSurda) [reply src/bitmessageqt/addressvalidator.py:8]: i think there should be a space here, maybe that's why it was complaining

File: src/bitmessageqt/addressvalidator.py (hunk 11 above)
Comment (PeterSurda) [reply src/bitmessageqt/addressvalidator.py:165]: Still `QT` and not `Qt`

Comment (coolguy-cell) [reply src/bitmessageqt/addressvalidator.py]: fixed.

File: src/bitmessageqt/addressvalidator.py [hunk 15]
```diff
@@ -1,14 +1,29 @@
-from PyQt4 import QtGui
+"""
+Address validator module.
+"""
+# pylint: disable=too-many-branches,relative-import,too-many-arguments,unused-argument,ungrouped-imports
+
 from Queue import Empty
+from PyQt4 import QtGui
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:7]: wrong order

File: src/bitmessageqt/addressvalidator.py [hunk 16]
```diff
@@ -60,25 +80,30 @@ def checkQueue(self):
...
-        if len(addressGeneratorReturnValue) == 0:
+        if not addressGeneratorReturnValue:
             self.setError(_translate("AddressValidator", "Address already present as one of your identities."))
             return (QtGui.QValidator.Intermediate, 0)
         if addressGeneratorReturnValue[0] == 'chan name does not match address':
-            self.setError(_translate("AddressValidator", "Although the Bitmessage address you entered was valid, it doesn\'t match the chan name."))
+            self.setError(
+                _translate(
+                    "AddressValidator",
+                    "Although the Bitmessage address you \
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:94]: Please no. Splitting strings into multiple lines should be like this:

```
    " aaaaa "
    " bbbb "
````

And there shouldn't be `\'` inside `""`.

**Comment by coolguy-cell** [comment]
Fixed.

**Comment by coolguy-cell** [comment]
fixed

File: src/bitmessageqt/addressvalidator.py [hunk 17]
```diff
@@ -1,14 +1,30 @@
+"""
+Address validator module.
+"""
+# pylint: disable=too-many-branches,relative-import,too-many-arguments
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:4]: `relative-import` not needed

File: src/bitmessageqt/addressvalidator.py [hunk 18]
```diff
@@ -1,14 +1,30 @@
...
+# pylint: disable=unused-argument,ungrouped-imports,wrong-import-order
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:5]: None of these are needed

**Comment by coolguy-cell** [comment]
Done

File: src/bitmessageqt/addressvalidator.py [hunk 19]
```diff
@@ -60,25 +80,30 @@ def checkQueue(self):
...
+        if not addressGeneratorReturnValue:
             self.setError(_translate("AddressValidator", "Address already present as one of your identities."))
             return (QtGui.QValidator.Intermediate, 0)
         if addressGeneratorReturnValue[0] == 'chan name does not match address':
-            self.setError(_translate("AddressValidator", "Although the Bitmessage address you entered was valid, it doesn\'t match the chan name."))
+            self.setError(
+                _translate(
+                    "AddressValidator",
+                    "Although the Bitmessage address you "
+                    "entered was valid, it doesn\'t match the chan name."))
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:95]: no `\'`, just `'`

File: src/bitmessageqt/addressvalidator.py [hunk 20]
```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
         if address is None:
             addressGeneratorQueue.put(('createChan', 4, 1, str_chan + ' ' + str(passPhrase), passPhrase, False))
         else:
-            addressGeneratorQueue.put(('joinChan', addBMIfNotPresent(address), str_chan + ' ' + str(passPhrase), passPhrase, False))
+            addressGeneratorQueue.put(
+                ('joinChan', addBMIfNotPresent(address),
+                 str_chan + ' ' + str(passPhrase), passPhrase, False))
```
Comment (PeterSurda) [inline src/bitmessageqt/addressvalidator.py:158]: I would use `format` here, I was told using `+` to join strings should be avoided.

**Comment by coolguy-cell** [comment]
fixed

File: src/bitmessageqt/addressvalidator.py [hunk 21]
```diff
@@ -122,23 +153,28 @@ def validate(self, s, pos):
         if address is None:
             addressGeneratorQueue.put(('createChan', 4, 1, str_chan + ' ' + str(passPhrase), passPhrase, False))
         else:
-            addressGeneratorQueue.put(('joinChan', addBMIfNotPresent(address), str_chan + ' ' + str(passPhrase), passPhrase, False))
+            addressGeneratorQueue.put(
+                ('joinChan', addBMIfNotPresent(address),
+                 "{} {}".format(str_chan, str(passPhrase)), passPhrase, False))
```
Comment (g1itch) [inline src/bitmessageqt/addressvalidator.py:158]: str() seems redundant

Comment (coolguy-cell) [reply src/bitmessageqt/addressvalidator.py]: @g1itch its done.

**Pull request merged**